package.json

```diff
@@ -92,7 +92,7 @@
 		"@hashicorp/localstorage-polyfill": "^1.0.14",
 		"@hashicorp/mktg-global-styles": "^4.2.3",
 		"@hashicorp/mktg-logos": "^1.3.1",
-		"@hashicorp/platform-analytics": "^0.6.1",
+		"@hashicorp/platform-analytics": "^0.8.0-canary-2022830143359",
```

Comment:
This fixes [Profile dropdown account settings open new window](https://app.asana.com/0/0/1203075873125168/f)

See [this PR](https://github.com/hashicorp/web-platform-packages/pull/96) for more details.